solutions/remotemonitoring/scripts/individual/deployment-configmap.yaml

```diff
@@ -25,49 +27,21 @@ data:
   # and resource type "Iot Hub" then "Shared access policies" menu.
   iothub.connstring: "{IoT Hub connection string}"
 
-
   # The connection string Document DB will be of format "AccountEndpoint={URI};AccountKey={Key};"
   # The key will be shown in CLI window as output when remote-cli command has finished running.
   # Alternatively it can be accessed from http://portal.azure.com under {myResourceGroupName}
   # and resource type "Azure Cosmos DB account" then "Keys" menu.
   docdb.connstring: "{DocumentDB connection string}"
 
-  azureblob.account: "{Azure Blob account name}"
-  azureblob.key: "{Azure Blob account key}"
-  azureblob.endpointsuffix: "{Azure Blob account endpoint suffix}"
   azureblob.connstring: "{Azure Blob connection string}"
```

Comment:
There are still some secrets in this file. Is it because of DS?

Reply:
Correct it is because of DS. This is the iteration of the PR before having Sushil's scripts to be able to clean up more as we did in the 3rd PR.